posthog/models/special_migration.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+
+# an enum, essentially
+class MigrationStatus:
+    NotStarted = 0
+    Running = 1
+    CompletedSuccessfully = 2
+    Errored = 3
+    RolledBack = 4
+
+
+class SpecialMigration(models.Model):
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["name"], name="unique name",)]
+
+    id: models.BigAutoField = models.BigAutoField(primary_key=True)
+    name: models.CharField = models.CharField(max_length=400, null=False, blank=False)
+    progress: models.IntegerField = models.IntegerField(null=False, blank=False, default=0)
```

Comment:
Nit: do we need this? Isn't this `progress == !status`?

Reply:
Progress is out of 100. It's for running migrations, so we know how far along they are. Migration definitions provide a function to check progress. e.g. 10000/25000 events moved to the new table